android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -417,5 +446,6 @@ constructor(
     const val TYPE_REFERENCE_DELIMITER = "/"
     const val CONFIG_SUFFIX = "_config"
     const val ICON_PREFIX = "ic_"
+    const val MANIFEST_PROCESSOR_BATCH_SIZE = 30
```

Comment:
do we need to optimize this?

Reply:
what did you have in mind?

Reply:
what did you have in mind for the optimization?

Reply:
nothing specific, I'm not sure if this is trivial and if we change this to 10 or 300 the performance won't be affected, or if it's something we can adjust to get performance gains, don't think we need to do anything for this PR, but if there's we think there's a >=50/50 chance we can get meaningful perf gains from adjusting this let's create an issue -- I didn't create an issue b/c I don't know if that's true, maybe any tweaks we do will be negligible to performance, what do you think?

Reply:
@ndegwamartin could we use a bundle transaction? So add all the items in a specific section of composition to a `transaction` bundle with GET requests methods. 

We would need to change the request from the app to POST.

Reply:
Kinda like what the SDK does while doing a sync-up.